apps/default.nix

```
@@ -0,0 +1,20 @@
+{ ... }: {
+  perSystem = { self', ... }: {
+    apps = {
+      tofu-validate = {
+        type = "app";
+        program = "${self'.packages.tofu-validate}/bin/tofu-validate";
```

```suggestion
        program = lib.getExe self'.packages.tofu-validate;
```

You can do this if you prefer. Same for other 2 `apps` below.